web/src/engine/attachment/src/pageContextAttachment.ts

```diff
@@ -236,7 +236,7 @@ export class PageContextAttachment extends EventEmitter<EventMap> {
     if(x instanceof x.ownerDocument.defaultView.HTMLTextAreaElement) {
       return true;
     } else if(x instanceof x.ownerDocument.defaultView.HTMLInputElement) {
-      if (x.type == 'text' || x.type == 'search') {
+      if (x.type == 'email' || x.type == 'search' || x.type == 'text' || x.type == 'url') {
```

Comment:
Part of me wonders if we should migrate this condition to a method on `Input`... perhaps `Input.isSupportedType(type: string): boolean`?  We are kind of WET re: the types.

Could then call that directly for the _entire_ condition here and replace the switch in the other spot.

Reply:
This, in turn, allows `Input` to self-document which types it supports, rather than having them prescribed / defined outside of the class responsible for the element's primary support.

Reply:
Good idea. Done.